tests/gpflow/models/test_variational.py

```diff
@@ -111,7 +111,7 @@ def test_reference_implementation_consistency():
         q_mean.reshape(-1), p_mean.reshape(-1), q_cov.reshape(-1), p_cov.reshape(-1)
     )
 
-    assert_allclose(univariate_KL - multivariate_KL, 0, atol=4)
+    assert_allclose(univariate_KL.squeeze(), multivariate_KL.squeeze(), atol=4)
```

Comment:
why squeeze?

Reply:
ah, because they had different shapes ((N) vs (N,1)) or something like that; the subtraction before worked because of broadcasting, but it seemed more appropriate to use assert_allclose as intended. note that this is only a self-consistency check of two different reference implementations inside the tests.